sklearn/cluster/_kmeans.py

```diff
@@ -1513,6 +1516,17 @@ class MiniBatchKMeans(KMeans):
         partition (if compute_labels is set to True). The inertia is
         defined as the sum of square distances of samples to their nearest
         neighbor.
+        
+    Counts_ : array like, shape (k,)
+        An array of samples that can be randomly reassigned if the existing
+        sample cluster is no longer valid or helpfulfor finding the centroids.
+
+    init_size_ : int, default=None
+        Number of samples to randomly sample for speeding up the initialization
```

Comment:
This is a copy paste of the doctring for `init_size`. You need update it explain the difference with `init_size`.